services/src/main/java/org/keycloak/storage/UserStorageManager.java

```diff
@@ -362,54 +344,44 @@ public int getUsersCount(RealmModel realm, Set<String> groupIds) {
     }
 
     @Override // TODO: missing storageProviders count?
```

Comment:
@hmlnarik should this be added in future or should I remove these comments?

Reply:
Good catch @vramik. These methods need to fulfil the contract given in javadoc, and thus need to count in the federated storages as well

Reply:
We should probably create a Jira for this, or should I add it in this PR?

Reply:
Rather the latter (this PR) please.

Reply:
> [EDIT] @mhajas has convinced me that this is not a straightforward cleanup because of requirements it has on handling e.g. imported users and those from the original storage. Let's keep this in a separate task

So let's create a jira for that and leave it as it is.

Reply:
Here is Jira: https://issues.redhat.com/browse/KEYCLOAK-16816